elro/hub.py

```diff
@@ -50,6 +51,12 @@ async def sender_task(self):
         await self.sync_scenes(0)
         await self.get_device_names()
 
+        logging.info("Waiting until all devices are retreived")
+        await trio.sleep(5)
```

Comment:
why 5s? Could we react to the actual event somehow?

Reply:
The problem faced here is that it is not known how many devices will be returned. So we can wait for the first device name to return, but then we still don't know how much devices will show up. The best thing to do imo, is to rerun this sync some times more often, but it is not necessary to do this every 30 seconds. It takes some time to implement that. Is it ok that this becomes an issue and it is resolved later?